src/Raven.Server/Documents/Indexes/Errors/FaultyIndexDefinition.cs

```diff
@@ -11,7 +11,8 @@ public class FaultyIndexDefinition : IndexDefinitionBaseServerSide<IndexField>
         private readonly IndexDefinition _definition;
 
         public FaultyIndexDefinition(string name, IEnumerable<string> collections, IndexLockMode lockMode, IndexPriority priority, IndexState state, IndexField[] mapFields, IndexDefinition definition)
-            : base(name, collections, lockMode, priority, state, mapFields, IndexVersion.CurrentVersion, definition.DeploymentMode, definition.ClusterState)
+            : base(name, collections, lockMode, priority, state, mapFields, IndexVersion.CurrentVersion, definition.DeploymentMode, definition.ClusterState,
+                definition.SourceItemKind.GetValueOrDefault())
```

Comment:
I'm feeling safe because of these changes  https://github.com/ravendb/ravendb/pull/16732/commits/b0a44920c507a30a93e9e788d58da66cc86f7225, but I may write a test for that